src/main/java/com/shaft/gui/element/ElementActions.java

```diff
@@ -701,34 +701,31 @@ public ElementActions keyPress(By elementLocator, Keys key) {
      */
     public ElementActions select(By elementLocator, String valueOrVisibleText) {
         ElementInformation elementInformation = ElementInformation.fromList(ElementActionsHelper.
-                identifyUniqueElement(DriverFactoryHelper.getDriver(), elementLocator));
+                identifyUniqueElement((WebDriver) DriverFactoryHelper.getDriver(), elementLocator));
```

Comment:
I need to know why you casting the WebDriver while it's not needed previously.

Reply:
Yeah I forgot about not using threadlocal driver version, so it's not really needed any more but it won't cause any issues, can be removed In the next PR if we're going with this one

Reply:
It's better do it now because we always tend to forged these things :D

Reply:
Casting removed 😄